roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -102,6 +102,11 @@
     delete_after: true
 
 # configmap
+- assert:
+    that:
+    - "{{ openshift_logging_elasticsearch_kibana_index_mode in __kibana_index_modes }}"
```

Comment:
do you need to have a string and a template evaulation here?  or can you just use
```
- openshift_logging_elasticsearch_kibana_index_mode in __kibana_index_modes
```
?